compiler/injectdestructors.nim

```diff
@@ -471,10 +472,9 @@ proc recurse(n: PNode, c: var Con, processProc: proc): PNode =
           branch.add processProc(n[i][0], c)
       result.add branch
   else:
-    assert(false, $n.kind)
```

Comment:
I got nodes of kind nkRefTy, nkRaiseStmt and nkTypeOfExpr occured here.

Reply:
I'd prefer handling them explicitly and leaving the assert in there